spec/unit/recipes/default_spec.rb

```diff
@@ -18,7 +18,7 @@
         case chef_run.node['platform_version'].to_i
         when 6
           %w[initscripts module-init-tools]
-        when 7
+        when 7..9
           %w[systemd kmod]
```

Comment:
Aren't you supposed to have systemd-udev here for el9 ?

Reply:
Indeed it's not consistent. Although, `rpm -q --whatprovides /etc/modules-load.d` on a RHEL8 host was already giving "systemd-udev", maybe we should patch it as well.

Reply:
Changed tests so they are accurate with RHEL9